src/kuzzle.js

```diff
@@ -1,9 +1,9 @@
 var
   uuid = require('uuid'),
-  KuzzleDataCollection = require('./kuzzleDataCollection'),
-  KuzzleSecurity = require('./security/kuzzleSecurity'),
-  KuzzleMemoryStorage = require('./kuzzleMemoryStorage'),
-  KuzzleUser = require('./security/kuzzleUser'),
+  Collection = require('./kuzzleCollection'),
```

Comment:
I'd rather leave the `kuzzle` prefix to the files, to keep this notion of namespacing we have in the code. But I've no strong opinion on it.